README.md

```diff
@@ -1,6 +1,6 @@
-# Parity - fast, light, and robust Ethereum client
+## Parity-Ethereum - a fast, light, and robust EVM and WASM blockchain client
 
-## [» Download the latest release «](https://github.com/paritytech/parity/releases/latest)
+### [» Download the latest release «](https://github.com/paritytech/parity/releases/latest)
```

Comment:
Don't we need to update this link as well to `https://github.com/paritytech/parity-ethereum/releases/latest`?

Reply:
We are not there yet :)

```
curl -I https://github.com/paritytech/parity-ethereum/releases/latest
HTTP/1.1 404 Not Found
```